examples/policies/l3/egress-allow-all/egress-allow-all.json

```diff
@@ -0,0 +1,9 @@
+[{
+    "labels": [{"key": "name", "value": "allow-all-from-victim"}],
+    "endpointSelector": {"matchLabels": {"role":"victim"}},
```

Comment:
This doesn't match the yaml selector.

Reply:
Time to pull out https://github.com/cilium/1abel?

Reply:
Will be fixed in latest upload.